controllers/apps/v1beta4/emqx_controller.go

```diff
@@ -121,6 +122,12 @@ func (r *EmqxReconciler) Do(ctx context.Context, instance appsv1beta4.Emqx) (ctr
 	sts = updateStatefulSetForACL(sts, acl)
 	sts = updateStatefulSetForPluginsConfig(sts, generateDefaultPluginsConfig(instance))
 	sts = updateStatefulSetForLicense(sts, license)
+
+	bootstrapUser := &corev1.Secret{}
```

Comment:
Needless this line

Reply:
done

Reply:
So why this line still here ？I didn't see any update